.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test_ale:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        vim-version:
+          - '--vim-80-only'
+          - '--vim-81-only'
+          - '--neovim-only'
+          - '--linters-only'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run tests
+        run: ./run-tests -v ${{ matrix.test }}
```

Comment:
Believe it should be `matrix.vim-version` instead.  Need to use the name give inside the strategy block.

Reply:
Fixed, thanks.